jdl/validators/jdl-without-application-validator.js

```diff
@@ -46,13 +46,22 @@ module.exports = {
  * @param {String} applicationSettings.applicationType - the application type.
  * @param {String} applicationSettings.databaseType - the DB type.
  * @param {Boolean} applicationSettings.skippedUserManagement - whether user management is skipped.
+ * @param {Object} applicationSettings.blueprints - the blueprints used.
```

Comment:
Is it an array or an object? I'm seeing it responds to `length`

Reply:
change to array